databases/week3/queries3.sql

```diff
@@ -0,0 +1,41 @@
+use meal_sharing;
+
+#Get meals that has a price smaller than a specific price fx 90
+SELECT * FROM meal 
+WHERE price < 90;
+
+#Get meals that still has available reservations
+SELECT meal_id, title, SUM(number_of_guests) AS number_of_guests, max_reservations FROM meal
+LEFT JOIN reservation ON meal.id= reservation.meal_id
+group by meal.id
+HAVING meal.max_reservations > number_of_guests OR number_of_guests IS NULL ;
```

Comment:
I would always select `meal.id` instead of `reservation.meal_id` to avoid NULL in the id column.
Also consider using the COALESCE() function to get a zero instead of NULL for `number_of_guests`. That way you can skip the `OR number_of_guests IS NULL`

See: https://dev.mysql.com/doc/refman/8.0/en/comparison-operators.html#function_coalesce